fast-af/fast-checkout-woocommerce
# Resolves PINT-619 - Add Freemius analytics

# Description

Add Freemius Analytics to collect information about active installs.

# Testing instructions

1. Login to the staging admin.
2. Visit the Fast plugin settings page.
3. Verify that the Freemius opt-in tool pops up.
4. Opt-in to Freemius.
5. Verify that you received an email from Freemius to verify your opt-in.

# Checks

@ilkerulutas @brikr this is ready for review

**Note:** Everything in the `freemius` folder is a third-party SDK from freemius.com. The only code that I added was to `/fast.php` and `/includes/class-freemius.php`.
